core/src/test/resources/sqllogictest/test/sql/aggregate/group/test_group_by.test

```diff
@@ -0,0 +1,190 @@
+# name: test/sql/aggregate/group/test_group_by.test
```

Comment:
nit: `test/sql/group_by.test` would be more consistent with the remaining tests.

Reply:
i copied this hierarchy from duckdb repo